### PR TITLE
Feature/pool 174 prizepool#redeemticketsinstantly unit

### PR DESCRIPTION
**Pull request by robsecord**

### CODE REVIEW

File: contracts/periodic-prize-pool/InterestTracker.sol [hunk 1]
```diff
@@ -51,6 +51,7 @@ abstract contract InterestTracker is AbstractYieldService {
   }
 
   function captureInterest() internal returns (uint256) {
+    // console.log("POKE!");
```
Comment (asselstine) [inline contracts/periodic-prize-pool/InterestTracker.sol:54]: ✂️

File: contracts/periodic-prize-pool/InterestTracker.sol [hunk 2]
```diff
@@ -81,6 +82,7 @@ abstract contract InterestTracker is AbstractYieldService {
 
   function poke() internal {
     uint256 unaccountedBalance = _unaccountedBalance();
+    // console.log("poke() unaccounted: %s", unaccountedBalance);
```
Comment (asselstine) [inline contracts/periodic-prize-pool/InterestTracker.sol:85]: ✂️

File: contracts/periodic-prize-pool/PeriodicPrizePool.sol [hunk 3]
```diff
@@ -386,39 +391,53 @@ abstract contract PeriodicPrizePool is Timelock, BaseRelayRecipient, ReentrancyG
     return FixedPoint.calculateMantissa(_balanceOfTicketInterest(user, tickets), tickets);
   }
 
-  function redeemTicketsInstantly(uint256 tickets, bytes calldata data) external nonReentrant returns (uint256) {
+  function redeemTicketsInstantly(
+    uint256 tickets, 
+    bytes calldata data,
+    bytes calldata operatorData
```
Comment (asselstine) [inline contracts/periodic-prize-pool/PeriodicPrizePool.sol:397]: This isn't an operator function, so operatorData isn't necessary here.

File: contracts/prize-strategy/SingleRandomWinnerPrizeStrategy.sol [hunk 4]
```diff
@@ -43,6 +43,7 @@ contract SingleRandomWinnerPrizeStrategy is Initializable, BaseRelayRecipient, S
     uint256 requestId = rngRequestIds[address(prizePool)];
     bytes32 randomNumber = rng.randomNumber(requestId);
     uint256 prize = prizePool.awardPrize();
+    console.log("PRIZE IS %s", prize);
```
Comment (asselstine) [inline contracts/prize-strategy/SingleRandomWinnerPrizeStrategy.sol:46]: ✂️

File: contracts/periodic-prize-pool/InterestTracker.sol (hunk 1 above)
Comment (robsecord) [reply contracts/periodic-prize-pool/InterestTracker.sol]: done

File: contracts/periodic-prize-pool/PeriodicPrizePool.sol (hunk 3 above)
Comment (robsecord) [reply contracts/periodic-prize-pool/PeriodicPrizePool.sol]: done

**Pull request merged**